README.md

```diff
@@ -1 +1,105 @@
-# angular-testing-filters-readme
\ No newline at end of file
+# Testing Filters
+
+## Overview
+
+Now that we can utilise the power of filters, let's test them to make sure they're returning the correct data.
+
+## Objectives
+
+- Describe filter testing
+- Write a unit test for filters
+
+## Setup tests
+
+As we learned earlier, we can use either filters in the DOM or in our controllers.
+
+We're going to be writing our filter's tests using Karma and Jasmine - this means we will have to be using the filters in our controllers as we cannot test our DOM.
+
+The tests we are about to write are quite similar to tests we've done before. We will be using a controller that filters a list, and we'll be ensuring that filtered list is correct.
+
+Let's take our basic controller. We've got a list of people, a search term and our filtered list. We've got a function to call to re-filter our list when the search term changes.
```

Comment:
I'm a little bit confused by the `changeFilter` function - when would this get called? Every time the search model is updated, or would the user press like a `search` button or something?